virt/virt-4-13-release-notes.adoc

```diff
@@ -42,6 +42,9 @@ To view the supported guest operating systems for {VirtProductName}, see link:ht
 [id="virt-4-13-new"]
 == New and changed features
 
+// CNV-29445 FIPS not supported in 4.13
+* {product-title} {product-version} is based on {op-system-base-full} 9.2. {op-system-base} 9.2 has not yet been submitted for FIPS validation. Red Hat expects, though cannot commit to a specific timeframe, to obtain FIPS validation for {op-system-base} 9.0 and {op-system-base} 9.2 modules, and later even minor releases of {op-system-base} 9.x. Updates will be available in link:https://access.redhat.com/articles/2918071[Compliance Activities and Government Standards].
```

Comment:
^^ I rehashed the last block of text because it indicates future language and it contained a confusing clause (the placement of "even" caused me confusion). How about

> If Red Hat obtains FIPS validation for {op-system-base} 9.0, {op-system-base} 9.2, and later minor versions of {op-system-base} 9, you can locate this information at link:https://access.redhat.com/articles/2918071[Compliance Activities and Government Standards].

Reply:
I copied the text from the OCP RN, so it needs to stay this way for consistency.